nerfstudio/process_data/metashape_utils.py

```diff
@@ -198,4 +206,4 @@ def metashape_to_json(
 
     summary.append(f"Final dataset is {len(data['frames'])} frames.")
 
-    return summary
+    return summary
```

Comment:
@Jordan-Pierce, I think you'll need to keep the line break at the end of the file to pass the automatic test that failed.

Reply:
Hi @cr333, apologies, first time making a PR to an outside org. You're referring to just a newline after the return to deal with the linter?